src/components/forms/PlanForm/tests/fixtures.ts

```diff
@@ -936,7 +936,7 @@ export const planFormValues2 = {
   fiStatus: 'A2',
   identifier: '356b6b84-fc36-4389-a44a-2b038ed2f38d',
   interventionType: 'FI',
-  jurisdictions: [{ id: '3952', name: '3952' }],
+  jurisdictions: [{ id: '3952', name: '' }],
```

Comment:
Why is this change desirable?

Reply:
Caused by the change you mentioned [above](https://github.com/onaio/reveal-frontend/pull/1612#discussion_r645735594)